* Added content for noticeboard.

MoMa666 opened the pull request:
* Added content for noticeboard.
* An object class was created for filling the layout Objects.java

Commit:
Persönliche Begrüßung funktioniert mit festgelegter User ID

Commit:
Added dummy text and non-functional button to ArchiveView.java; edited comments to ArchiveView and SideBar

Commit:
Added files from Monika header change

Commit:
Added path for background-image in LoginView.java

sabrine-gamdou committed:
Created a page service class for all pages.

sabrine-gamdou committed:
Added the new pageService to the aboutUsView.

sabrine-gamdou committed:
Added the new pageService to the WelcomeView.

sabrine-gamdou committed:
updated header comment.

sabrine-gamdou committed:
updated header comment.

sabrine-gamdou committed:
deleted not needed ourCompany services.

MoMa666 committed:
* Added content for noticeboard.
* An object class was created for filling the layout Objects.java

sabrine-gamdou committed:
added missing notifications list of the user in the userEntity class.

sabrine-gamdou committed:
updated header data

sabrine-gamdou committed:
updated DocumentEntity class

sabrine-gamdou committed:
updated header comment

sabrine-gamdou committed:
updated NotificationEntity class.

sabrine-gamdou committed:
updated NotificationEntity class.

sabrine-gamdou committed:
updated header data.

sabrine-gamdou committed:
updated pageEntity.

sabrine-gamdou committed:
updated header comment.

sabrine-gamdou committed:
updated entity class of images.

sabrine-gamdou committed:
updated header comment.

sabrine-gamdou committed:
added find method to the ImageRepository

sabrine-gamdou committed:
added find method to the DocumentRepository.

sabrine-gamdou committed:
added pdfViewerClass, responsible for print the pdfs to the page.

sabrine-gamdou committed:
updated header comment.

sabrine-gamdou committed:
updated header comment.

sabrine-gamdou committed:
updated imports.

sabrine-gamdou committed:
This is class represents a pdf gui component, linking a documentEntity with
with a delete button and a pdfViewer component.

sabrine-gamdou committed:
created service class for documents.

sabrine-gamdou committed:
created service class for images.

sabrine-gamdou committed:
updated header comments.

sabrine-gamdou committed:
This is class represents a pdf gui component, linking a documentEntity with a delete button and a pdfViewer component.

sabrine-gamdou committed:
updated version

sabrine-gamdou committed:
This is class that takes care of deleting documents on the database and on the server.

sabrine-gamdou committed:
updated header comment.

sabrine-gamdou committed:
updated setter und getter of NotificationEntity.

sabrine-gamdou committed:
This is class manages the upload and pdf components and uses the
pdfCreationManager and the pdfDeletionManager to create and delete documents.

sabrine-gamdou committed:
This class manages the pdfs list and the pdfManagers.

sabrine-gamdou committed:
Updated entities that JPA doesn't like :D

sabrine-gamdou committed:
Deleted redundant getters and setters

anastasiya-jh commented on src/main/java/com/example/application/ui/horizontal/community/Objects.java:
```diff
@@ -0,0 +1,98 @@
+
+public class Objects implements Cloneable {
+    private int id;
+    private String cate;
```
cate as in category? Please use better names so others can understand better what the variable stands for. Please change the name

anastasiya-jh reviewed (changes requested):
NoticeBoardView.java: Tabelle muss alles auf deutsch stehen. Hier werden außerdem zu viele Styles genutzt.... bitte einmal alle Styles in die vorhergesehene .css Datei schieben. 
Objects.java wird später nicht mehr genutzt, sobald das backend implementiert wird. Deswegen haben wir hier jetzt nicht so stark auf die Benennung geachtet.

sabrine-gamdou committed:
Updated page entity, it now has pages.

sabrine-gamdou committed:
Updated imports.

sabrine-gamdou committed:
This class represents an image component, it receives an imageEntity and the files input stream.

sabrine-gamdou committed:
This is an image viewer component

Commit:
Added background image and logo to LoginView.java

sabrine-gamdou committed:
This is class that takes care of creating the image on the
database and on the server.

sabrine-gamdou committed:
This is class that takes care of deleting images on the
database and on the server.

sabrine-gamdou committed:
This is class manages the upload and image components and uses the
imageCreationManager and the imageDeletionManager to create and delete images.

sabrine-gamdou committed:
This class manages the images list and the imageManagers.

sabrine-gamdou committed:
Set maximal size for uploaded files.

sabrine-gamdou committed:
Changed fetch type of the documents list in pageEntity.

rschiri committed:
added the links in BlogView and WikiView which open new tabs

Commit:
Added comments after change request

anastasiya-jh committed:
Updated Comments in HorizontalBar.java

MoMa666 committed:
* Request edited

Commit:
persönliche Begrüßung mit hart codierter UserID

anastasiya-jh commented on src/main/java/com/example/application/ui/horizontal/community/NoticeBoardView.java:
```diff
@@ -1,28 +1,136 @@
  *  @version 1.0
- *  @since 15.12.2020
- *  @lastUpdated
+ *  @since 19.01.2021
```
Uhm @ since wird hier nie geändert. Das Datum wird immer das gleiche sein

anastasiya-jh reviewed (changes requested):
Wir warten darauf, bis das @ since wieder zurückgeändert wird und der Tabellen-Header auf deutsch ist und Die Textzeile über der Tabelle ist  :-)

rschiri committed:
changed the button for light/dark mode to a switch in SettingsView

MoMa666 committed:
Ich musste nochmal die aktuelle main ziehen und den logincontroller neu einfügen.
Desweiteren habe ich schonmal alle Übersichtsseiten in der horizontalen mit den festgelegten page ids auslesen lassen

MoMa666 committed:
LibraryView page mit db verknüpft

MoMa666 committed:
die LoginController klasse wird nicht mehr extra benötigt der Zugriff auf username wird im welcomeView aufgerufen und dort abgegelichen. Somit wurde der Fehler der throw execption mit einer Abfrage behoben und man kann sich nun auch ohne Probleme wieder ausloggen.

MoMa666 committed:
Ich habe bis auf die Notifications und Ideenmanagement alle page über die in der Db hinterlegten ID angebpasst so das nun auf diesne Seiten der Titel und der Text daraus gelesen wird.
* Ich habe die losen Objekte in ein Layout eingefügt oder in einen Span oder Div

MoMa666 committed:
Ich habe eine Klasse für den GetUserName erstellt und die Funktion so angepasst das sie diesen als String retuned. Somit muss nur noch die Klasse erzeugt werden auf den Seiten in denen man auf den aktuellenUsername aus dem authentifikator zugreifen möchte und kann ihn dann in die UserEntity klasse mit findUsername nutzen.

MoMa666 committed:
In myProfile wird der aktuelle User jetzt nicht mehr hardcodiert übergeben sondern durch den UserController der den angemeldeten User zurück gibt.MyProfile müsste nun somit komplett aus der datenbank gelesen werden können und auch in die datenbank schreiben
